.github/workflows/py-publish.yml

```diff
@@ -0,0 +1,27 @@
+name: Upload Python Package
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    permissions:
+        id-token: write
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.x"
+          cache: "pip"
+      - name: Install poetry
+        run: curl -sSL https://install.python-poetry.org | python3 -
+      - name: Install dependencies
+        run: |
+          poetry install
+      - name: Build and Publish package
+        run: |
+          poetry config pypi-token.pypi ${{ secrets.your-api-token }}
```

Comment:
this still needs to be updated @gabriel-tessier

Reply:
@tb102122 
I submitted an organization called diagrams-web on pypi it's still in pending state from more than 2 weeks.
I didn't want to publish it on my own account and transfer it later as I want to have the package on the organization from the beginning.


**diagrams-web Community Request Submitted**

**Submitted Jul 13, 2024**

 
But as it take more time than I expected for Pypi to approve, I will publish it on my own account.
About the py-publish.yml, let's keep this as it and wait to have the information for the organization to set teh secrets.

You will receive an invitation for the organization as soon as it get approved.